app/assets/json/translations/react/en.json

```diff
@@ -1,174 +1,217 @@
 {
```

Comment:
I don't think we want the wider space here so that the formatting matches all of our other files

Reply:
This is driven by [.editorconfig](https://github.com/SFDigitalServices/sf-dahlia-web/blob/5accc6220f1b28eea3f6545547b4111abdedd6b4/.editorconfig#L15-L15) Do we want to change this configuration?